tests/src/system/basic/WskBasicTests.scala

```diff
@@ -362,6 +372,25 @@ class WskBasicTests
             wsk.rule.list().stdout should include(ruleName)
     }
 
+    it should "display a rule summary when --summary flag is used with 'wsk rule get'" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val ruleName = "mySummaryRule"
+            val triggerName = "summaryRuleTrigger"
+            val actionName = "summaryRuleAction";
+            assetHelper.withCleaner(wsk.trigger, triggerName) {
+                (trigger, name) => trigger.create(name)
+            }
+            assetHelper.withCleaner(wsk.action, actionName) {
+                (action, name) => action.create(name, defaultAction)
+            }
+            assetHelper.withCleaner(wsk.rule, ruleName, confirmDelete = false) {
+                (rule, name) =>
+                    rule.create(name, trigger = triggerName, action = actionName)
+            }
+            val stdout = wsk.rule.get(ruleName, summary = true).stdout
+            stdout should include regex (s"(?i)rule\\s+/guest/${ruleName}")
```

Comment:
Indeed - good point. how about this: `ns` `=` `wsk namespace list` `toSet`, the modify should include to check that at least one match works for every space in the set?